### ISSUE TITLE
FontValidator exits with 1 (instead of zero) on a successful run

### ISSUE THREAD

**Issue by erniemarch2** [HinTak/Font-Validator#48]
When called from the command line (python script looping through several fonts in a folder),
font validator processes the first item and returns non-zero exit status of 1, causing
the python script to stop and not process the remaining items.

**Comment by HinTak** [comment]
You are aware that fontval itself takes multiple font arguments? As in '... -file font1.ttf -file font2.ttf ...'?

Also, if you really want to not stop on fatal errors, you do 

'find /dir -type f -name "*.ttf" -exec FontValidator ... {} \;' ?

This is what I do for testing all 3500 fonts that fedora ships.

**Comment by miguelsousa** [comment]
> You are aware that fontval itself takes multiple font arguments? As in '... -file font1.ttf -file font2.ttf ...'?

Yes, but we can't do that because the process we're running needs to call other tools on the same font.

> Also, if you really want to not stop on fatal errors, you do
> 'find /dir -type f -name "*.ttf" -exec FontValidator ... {} ;' ?

I appreciate the suggestion, but we're not running FontValidator from the command-line, we're calling it from a Python subprocess.

**Comment by HinTak** [comment]
The exit status from mono when FontValidator is running outside of it seems to be correct, and it looks like a mono bug for the process of bundling mono with FontValidator. I'll file a bug upstream.

**Comment by miguelsousa** [comment]
@HinTak thank you 🙇

**Comment by HinTak** [comment]
I did file it upstream, https://github.com/mono/mono/issues/13629 , just forgotten to come back to mention it here. Feel free to ping upstream...
